pegasus/sites.v3/code.org/views/header2.haml

```diff
@@ -40,9 +40,7 @@
 
       #right
         #sign_in_or_user{:style=>'display:block; float: left; margin: .6875em 0 0 30px;'}
-        - hamburger_class = (user_type == "student" || user_type == "teacher" ? "" : "mobile-feature")
-        %span{class: hamburger_class}
-          = view :hamburger, hamburger_options
+        = view :hamburger, hamburger_options
```

Comment:
Why don't you need the hamburger_class anymore? 
Because the hamburger has some other way to know about the user_type?